### PR TITLE
Feature/donate flow

### PR DESCRIPTION
**Pull request by abdiG**
created component/hourGlass.js
modified static/styles/main.scss
pages/account.js
created static/styles/partials/_hourglass.scss
please let me know if i can improve

### CODE REVIEW

File: static/styles/main.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@
 @import 'partials/_images.scss';
 @import 'partials/_account.scss';
 @import 'partials/_success.scss';
+@import 'partials/hourglass';
```
Comment (stripedpajamas) [inline static/styles/main.scss]: This should be `partials/_hourglass.scss` right?

Comment (abdiG) [reply static/styles/main.scss]: Ohh im sorry. Yes that was meant to be '@import `partials/hourglass'.scss;`

**Comment by stripedpajamas** [comment]
Ok update that with the proper file name. Should be `partials/_hourglass.scss;`.

File: static/styles/main.scss [hunk 2]
```diff
@@ -21,6 +21,7 @@
 @import 'partials/_images.scss';
 @import 'partials/_account.scss';
 @import 'partials/_success.scss';
+@import 'partials/hourglass.scss';
```
Comment (stripedpajamas) [inline static/styles/main.scss]: @abdiG I still don't think this is right -- it looks like the filename you created is `_hourglass.scss` with an underscore, but this import doesn't have the `_`.

**Comment by stripedpajamas** [comment]
Hi @abdiG -- I recently spoke to the content designer about the account page and it turns out that the hourglass idea is no longer needed. Really appreciate you helping out and putting in this PR, sorry for the wasted time!

**Pull request closed without merging**